source/includes/administration/_organizations.md

```diff
@@ -143,11 +145,15 @@ Attributes | &nbsp;
 
 `POST /organizations`
 
-Creates a new organization as a sub-organization of the caller's organization, or a sub-organization of the specified `parent`. The caller requires the `Organizations create` permission.
+Creates a new organization as a sub-organization of the caller's organization, or a sub-organization of the specified `parent` or at the top level if the calling user has an unscoped permission. The caller requires the `Organizations create` permission.
+
+Any service connections that are supplied in the request will be assigned to the organization asynchronously, The state of the connections assigned to the organization and its progress is reflected in the organization service connections.
```

Comment:
`organization asynchronously,` ---> `organization asynchronously.` (comma to period)